features/FxA-89-devices-view/README.md

```diff
@@ -411,6 +409,17 @@ Acceptance criteria:
       by accessing the devices view on a desktop browser,
       and then go to the sync preferences on the mobile device,
       I immediately see that the device is disconnected.
+* [ ] - Moved from Phase 2: When viewing my device list
```

Comment:
> Do we want a user to be able to get into a state where there is
> no service or device connected to their account?

It's not a state we *want* them to be in, but it's entirely possible.  Let's see if we can get some metrics on it.